src/components/site/utilities/TranslatedValue/index.jsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { useTranslation } from 'gatsby-plugin-react-i18next';
+import { isArray } from 'lodash';
+import { getValue, addTheCommas, formatValue } from '../../../../lib/utilities';
+
+const TranslatedValue = ({ accessor, data }) => {
```

Comment:
mirror of the `getValue` utility but utilizing React and adding support for translations. Will fall back on `getValue` using the same logic that `getValue` uses to fall back on returning the raw data.